### PR TITLE
VueUI viewer for large lists in VV

### PR DESCRIPTION
**Pull request by JohnWildkins**
implements a vue-based viewer for lists
shows you the contents of the list in raw html

i put about 19 seconds into this and tested it exactly once

i will not elaborate further and will take no questions

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!review

File: vueui/src/components/view/admin/extended-list.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <table>
+      <ul v-for="listitem in listvar" :key="listitem.key">
```
Comment (Karolis2011) [inline vueui/src/components/view/admin/extended-list.vue:4]: v-bind key is with totally invalid value

**Review by Karolis2011 — CHANGES_REQUESTED** [review]
This in theory doesn't handle associative lists

File: vueui/src/components/view/admin/extended-list.vue (hunk 1 above)
Comment (JohnWildkins) [reply vueui/src/components/view/admin/extended-list.vue]: yeah i forgot how to code for a second, happens to the best of us

and also me

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**